docs/fundamentals/flow/flow-api.md

```diff
@@ -230,6 +231,58 @@ executors:
 :align: center
 ```
 
+## GraphQL Interface
+
+````{admonition} See Also
+:class: seealso
+
+This article does not serve as the introduction to GraphQL.
+If you are not already familiar with GraphQL, we recommend you learn more about GraphQL from the official [GraphQL documentation](https://graphql.org/learn/).
+You may also want to learn about [Strawberry](https://strawberry.rocks/), the library that powers Jina's GraphQL support.
+````
+
+Jina Flows that use the HTTP protocol provide a GraphQL API out of the box, which is located behind the '/graphql' endpoint.
+
+This means that you can access the Flow from any GraphQL client, like for example, `sgqlc`.
```

Comment:
I would also explain here the benefit. About limiting the schema or defining the schema. Compare the result with one or the other I'd say?